src/client/src/orangehrmClaimPlugin/components/EditExpenseModal.vue

```diff
@@ -0,0 +1,212 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <oxd-dialog @update:show="onCancel">
+    <div class="orangehrm-modal-header">
+      <oxd-text type="card-title">
+        {{ $t('claim.edit_expense') }}
+      </oxd-text>
+    </div>
+    <oxd-divider />
+    <oxd-form :loading="isLoading" @submit-valid="onSave">
+      <oxd-form-row>
+        <oxd-grid :cols="1" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <claim-expense-type-dropdown
+              v-model="selectedOption"
+              :label="$t('claim.expense_type')"
+            ></claim-expense-type-dropdown>
+          </oxd-grid-item>
+        </oxd-grid>
+      </oxd-form-row>
+
+      <oxd-form-row>
+        <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <date-input
+              v-model="expense.date"
+              :label="$t('general.date')"
+              :rules="rules.date"
+              :years="yearsArray"
+              required
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
```

Comment:
no required

Reply:
@DulsaraNethmin no i meant `required` prop is not set. if it's not set the red asterix mark will not show in the field